Formula/taskcluster.rb

```diff
@@ -0,0 +1,26 @@
+class Taskcluster < Formula
+  desc "CI for Engineers"
+  homepage "https://taskcluster.net"
+  version "v44.17.2"
+  license "MPL-2.0"
+
+  if Hardware::CPU.physical_cpu_arm64?
```

Comment:
Homebrew also runs on linux: https://docs.brew.sh/Homebrew-on-Linux

I've seen quite a lot of installation pages that list this option. I think we can also add a check for mac/linux and fetch linux version here this way also.

Also homebrew does not support 32bit systems, so I think we don't have to worry about it for now

Reply:
Ah very cool, I had no idea! I'll add this in.

Reply:
Added in https://github.com/taskcluster/homebrew-tap/pull/1/commits/aa435528595a75e1761ab1f4a3916a43e7881729.